src/main/resources/org/sonar/l10n/openapi/rules/openapi/format/OAR113.html

```diff
@@ -0,0 +1,145 @@
+<p>Field or extension must be at the assigned location</p>
+<h2>Compliant Solution (OpenAPI 2)</h2>
+<pre>
+    swagger: "2.0"
+    info:
+      version: "1.0.0"
+      title: "Sample API"
+      x-custom-example: "example value"
+    paths:
+      /example:
+        get:
+          description: "Get example"
+          responses:
+            "200":
+              description: "Successful response"
+              schema:
+                $ref: "#/definitions/ExampleObject"
+    definitions:
+      ExampleObject:
+        type: object
+        properties:
+          id:
+            type: string
+          name:
+            type: string
+        x-extension-field: "example"
+</pre>
+<h2>Noncompliant Solution (OpenAPI 2)</h2>
+<pre>
+    swagger: "2.0"
+    info: # Noncompliant {{OAR113: Field or extension x-custom-example must be at the assigned location}}
```

Comment:
Validation parts should not be in the html documentation, it only makes sense at test level